dist/index.html

```diff
@@ -0,0 +1,180 @@
+                    </li>
+                    <li class="header__item">
+                        <a href="#" class="header__link">Pricing</a>
+                    </li>
+                    <li class="header__item">
+                        <a href="#" class="header__link">Contact</a>
+                    </li>
+                </ul>
+            </div>
+
+            <!-- Menu Buttons -->
+            <div class="header__menu-btns">
+                <div class="btn">Login</div>
+                <div class="btn btn--colored btn--big">
+                    <span>Join Us </span>
+                    <span class="btn__icon">
+                        <img src="./images/Vector.png" alt="">
+                    </span>
+                </div>
+            </div>
+        </nav> 
+        <section class="header__body">
+            <div class="header__content">
+                <span class="sub-title">Welcome</span>
+                <h4 class="header__text">A Great Place to <br> Receive Care</h4>
+                <h6 class="header__sub-text">Get your best looking smile now</h6>
+                <div class="header__btns">
+                    <div class="btn btn--colored">
+                        <span>Join Us </span>
+                    </div>
+                    <div class="btn btn--bordered">Learn More </div>
+                </div>
+            </div>
+            <div></div>
+        </section>
+    </header>
+    <main>
+        <section class="experts_page">
+            <div class="experts-page__img-holder">
+                <img class="experts-page__img" src="./images/1-removebg-preview.png" alt="experts image">
+            </div>
+            <div class="experts-page__message">
+                <div class="experts-page__border"></div>
+                <div class="title">Meet our Experts</div>
+                <div class="page-text page-text--left">Problem Trying to resolve the conflict
+                    between The two major realms of classical physics: Newtonian mechanics.</div>
+                <div class="experts-page__btn">
+                    <span>Load More </span>
+                    <box-icon class="experts-page__icon" name='chevron-right' color="#1BA0AB"></box-icon>
+                </div>
+            </div>
+        </section>
+        <!-- Team page start -->
+        <section class="team-page">
+            <div class="team-page__content">
+                <div class="sub-title sub-title--team">Team</div>
+                <div class="title">Our Team</div>
+                <div class="page-text text--start">Problem Trying to resolve the conflict
+                    between<br>
+                    The two major realms of classical physics: Newtonian mechanics.</div>
+            </div>
+            <div class="card-container">
+                <div class="card">
+                    <div class="card__img">
+                        <img src="./images/tean/user-cover-1.png" alt="">
```

Comment:
I have seen you have used HTML semantics on other parts of your code, it would be better if you do the same on image and provide the alt for all your images in case a user can't see your image the alt can provide information to that missing image.